types/redux-form/lib/Field.d.ts

```diff
@@ -17,7 +17,7 @@ export type Parser = (value: any, name: string) => any;
 export type Validator = (value: any, allValues?: any, props?: any, name?: any) => any;
 
 export type EventHandler<Event> = (event: Event) => void;
```

Comment:
Could you please also consider updating the `EventHandler`, as both `onFocus` and `onDragStart` event handlers receive field's `name` as a second argument?

```suggestion
 export type EventHandler<Event> = (event: Event, name?: string) => void;
```